app/app.html

```diff
@@ -401,8 +412,15 @@
               <input class="name-input" value="<%= name %>">
             </li>
 
-          </ul>
-        </div>
+            <li class="dropdown-header shown-items hidden">Shown array items</li>
+
+            <li class="node-dropown-menu-item shown-items hidden">
+                <input type="number" class="shown-items" min="1" max="10" value="10" step="1">
+                <span class="shown-items" style="padding-left: 5px">of 1 is shown</span>
+            </li>
+
```

Comment:
this part of the config section will be hidden if the node doesn't have array items (not array or empty array)